ui/components/app/detected-token/detected-token-selection-popover/detected-token-selection-popover.js

```diff
@@ -18,11 +20,23 @@ const DetectedTokenSelectionPopover = ({
   setShowDetectedTokens,
 }) => {
   const t = useI18nContext();
+  const trackEvent = useContext(MetaMetricsContext);
 
   const detectedTokens = useSelector(getDetectedTokensInCurrentNetwork);
+  const detectedTokensDetails = detectedTokens.map(
+    ({ address, symbol }) => `${symbol} - ${address}`,
+  );
 
   const onClose = () => {
     setShowDetectedTokens(false);
+    trackEvent({
+      event: '"Import Tokens" cancelled',
```

Comment:
I'm not sold on using quotations around the Import Tokens here or in the Clicked one. can you confirm the source of this so I can explore?

Reply:
That was a mistake on my part; but I do have a small query, the popover we are opening and canceling on click are actually `Tokens found`  not the `Import Token` page; so should we have the event to something like `Clicked Tokens found` and `Cancelled Tokens found`?

Reply:
To avoid interchanging synonyms, I suggest incorporating "detect" over "found" here. I think sometimes we can get away with this if it makes sense to present synonyms to the user. Maybe we could update these to "Clicked Detected Tokens" and "Canceled Detect Tokens". 

In regards to the quotes in the event name, we do have some existing examples with this pattern:

- `/ui/components/app/import-token-link/import-token-link.component.js`
  - 'Clicked "Add Token"'
- `/ui/pages/swaps/fee-card/fee-card.js`:
  - 'Clicked "Gas Fees: Learn More" Link'
  
I'm personally okay with the quotes. I could see how it can be easier to read with quotes. It could cause some adjustable complications if we changed the copy, but I also don't imagine the copy would be changed often.

Reply:
New information found: 

John [shared a doc](https://docs.google.com/document/d/1WQ3WzRGamAAtO8g3JtQnztjxV8B-EiVK4DcjkDUY1co/edit#heading=h.qnasv64mcgjm) in [our metrics Slack channel](https://consensys.slack.com/archives/G01FXS64DFA/p1651014166856429),  which proposes either the object-action or context-object-action naming convention. We have some instances in our app that already use this convention e.g. 'Token Added'. 

More information on the object-action naming convention here: 
[ Segment naming convention for clean data article](https://segment.com/academy/collecting-data/naming-conventions-for-clean-data/) 

I have updated the following suggestions in my comment above this with this convention.

Reply:
The two prior examples containing quotations are from our legacy schema, so let's avoid those. 

I want to avoid using the word "Detect" in this situation so it doesn't get confused with Token Detected. 

The user is shown a phrase, "X tokens detected, click here to import them" or something to that effect.

Maybe we should get John to weigh in on this. I"ll start a thread in metametrics channel